lib/i18n/en.js

```diff
@@ -295,4 +297,28 @@ export default {
     outofdate: 'In order to continue using IRMA, you will need to update this app. You can update this IRMA app from the app store where you originally obtained it.',
     update: 'Update',
   },
+  LogDashboard: {
+    title: 'History',
+    issuing: {
+      title: '{{attr}} issued',
+      serverName: 'by {{serverName}}',
+      header: 'Attributes received',
+    },
+    disclosing: {
+      title: '{{attr}} disclosed',
+      serverName: 'to {{serverName}}',
+      header: 'Attributes disclosed',
+    },
+    signing: {
+      title: 'Signed message and {{attr}} disclosed',
+      serverName: 'to {{serverName}}',
+      header: 'Signed message',
+    },
+    removal: {
+      title: '{{attr}} removed from the app by you',
+    },
+    atTime: 'at {{time}}',
+    noLogs: 'History is empty yet',
```

Comment:
I think we should leave out the 'yet'. In future you can perhaps also clear your history (e.g. when you delete your account), so it would become empty again.

Reply:
Yeah okay, good point. Now this line is not visible at all, because in principle you always get the MijnIRMA credential and therefore it is never empty. It is mainly for the future.